src/NUnitTestAdapterInstall/packages.config

```diff
@@ -1,5 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Mono.Cecil" version="0.9.6.1" targetFramework="net45" />
+  <package id="Mono.Cecil" version="0.9.6.1" targetFramework="net35" />
```

Comment:
We'll need to be careful that this doesn't get toggled back if we ever update the version.